python/python/glide/async_commands/command_args.py

```diff
@@ -63,3 +63,34 @@ class ListDirection(Enum):
     """
     RIGHT: Represents the option that elements should be popped from or added to the right side of a list.
     """
+
+
+class ObjectType(Enum):
```

Comment:
Same comment as above
Here is a bit different since it can be used in other places, but since i need the user to use i'll break the command. 
Ideas? Create an empty class inherit from a class?